config/main.py

```diff
@@ -1375,7 +1375,7 @@ def memory(kdump_memory):
         config_db.mod_entry("KDUMP", "config", {"memory": kdump_memory})
         run_command("sonic-kdump-config --memory %s" % kdump_memory)
 
-@kdump.command()
+@kdump.command('num_dumps')
```

Comment:
this command hasn't been released. so, i suggest change to num-dumps

Reply:
Changed.